dsgrid/dimension/standard.py

```diff
@@ -28,23 +29,65 @@
 class CensusDivision(GeographyDimensionBaseModel):
     """Census Region attributes"""
 
+    time_zone: Optional[TimeZone] = Field(
```

Comment:
Do you want this type to allow a value of `None`? Since you are using `Optional` and a default value, `None` is a valid value that the user can pass.
```
time_zone: Optional[TimeZone] = Field(default=TimeZone.UTC)  # User can pass None
time_zone: TimeZone = Field(default=TimeZone.UTC)  # Only TimeZone values are allowed
time_zone: Optional[TimeZone] = Field()  # Default is None, User can also pass TimeZone values
```
Would `None` make sense as the default if you defined this field in `GeographyDimensionBaseModel`?